src/build/optimize-streams.ts

```diff
@@ -158,6 +180,9 @@ export function getOptimizeStreams(optimizeOptions: OptimizeOptions) {
   if (optimizeOptions.jsMinify) {
     streams.push(gulpif(/\.js$/, new JSOptimizeStream({fromString: true})));
   }
+  if (optimizeOptions.jsCompile) {
+    streams.push(gulpif(/\.js$/, new JSBabelStream({presets: [babelPresetES2015NoModules]})));
```

Comment:
@FredKSchott because people will want to have control over the preset, why we don't use `babel-env` preset instead of `es2015` preset? This make flexible the build and people don't need to create custom builds because they want to use any hot feature that is not on `es2015`?